packages/devtools/client/modern.config.ts

```diff
@@ -1,69 +1,29 @@
-import { URL } from 'url';
-import path from 'path/posix';
-import { execSync } from 'child_process';
-import { logger } from '@modern-js/builder-shared';
 import { appTools, defineConfig } from '@modern-js/app-tools';
 import { proxyPlugin } from '@modern-js/plugin-proxy';
 import { version } from './package.json';
 
-const commitShort = execSync('git rev-parse --short=10 HEAD').toString().trim();
-if (!commitShort.match(/^\w{10}$/)) {
-  throw new Error("Can't resolve git commit hash.");
-}
-
-const basename = new URL(process.env.DEPLOY_HOST || 'https://modernjs.dev');
-if (process.env.BASENAME === 'version' || !process.env.BASENAME) {
-  basename.pathname = `/devtools/${version}`;
-} else if (process.env.BASENAME === 'commit') {
-  basename.pathname = `/devtools/${commitShort}`;
-} else if (process.env.BASENAME === 'false') {
-  basename.pathname = '/devtools';
-} else {
-  basename.pathname = path.resolve('/devtools', process.env.BASENAME);
-}
-
-logger.info(
-  `Access client:`,
-  `${basename.href}?src=ws://localhost:8080/_modern_js/devtools/rpc`,
-);
-
 // https://modernjs.dev/en/configure/app/usage
 export default defineConfig<'rspack'>({
   runtime: {
     router: {
-      basename: basename.pathname,
+      basename: '/_modern_js/devtools',
```

Comment:
The `'/_modern_js/devtools'` is used many times, it can be extracted as a constant

Reply:
okk. btw the basename is configurable at first. But when I meet some difficulties due to using dynamic assetPrefix configuration, I had to change it to a fixed value with the `/_modern_js` prefix. Do you have any ideas on how to support different framework's route prefixes (such as `/_edenx`)?

Reply:
No idea for this...Maybe just `/__devtools`

Reply:
okk